frontend/src/components/widgets/FileUploader/FileUploader.scss

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright 2018-2019 Streamlit Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "src/assets/css/variables";
+
+.stFileUploader button:focus {
+  outline: 0;
+}
+
+.stFileUploaderError {
```

Comment:
We only add `st` to the top-level classname in each component.

Also, wrap the whole CSS in `.stFileUploader {` and then you can just rename this to `.uploadError` or something. For example:

```scss
.stFileUploader {
  ...
  .uploadError {
    ...
  }
  ...
}
```
...same with everything else in this file.

That's what we've been doing in the CSS for other elements.